main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        cat
+//  action 1:    the cat runs
+cat.run();
+//  action 2:    the cat meows
+cat.meow();
+//  action 3:    the cat eats
+cat.eat();
+//  2)            
+//  Noun:        vinylRecord
+//  action 1:    the vinylRecord plays music
+vinylRecord.playsMusic();
+//  action 2:    the vinylRecord skips
+vinylRecord.skip();
+//  action 3:    the vinylRecord turns
+vinylRecord.turn();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        weightLifter
+//  action 1:    the weightLifter lifts weights
+weightLifter.liftsweight();
+//  action 2:    the weightLifter sweats
+weightLifter.sweat();
+//  action 3:    the weightLifter does pushups
+weightLifter.doespushup();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        chef    
+//  action 1:    the chef cooks
+chef.cook();
+//  action 2:    the chef cuts onions
+chef.cutOnion();
+//  action 3:    the chef writes recipes 
+chef.writeRecipe();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        leg
+//  action 1:    the leg kicks
+leg.kick();
+//  action 2:    the leg stretches 
+leg.stretch();
+//  action 3:    the leg twists
+leg.twist();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        musician
+//  action 1:    the musician performs
+musician.perform();
+//  action 2:    the musician composes
+musician.compose();
+//  action 3:    the musician practices
+musician.practice();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        plant
+//  action 1:    the plant grows
+plant.grow();
+//  action 2:    the plant dies
+plant.die();
+//  action 3:    the plant gives fruit
+plant.giveFruit();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        athlete  
+//  action 1:    the athlete competes
+athlete.compete();
+//  action 2:    the athlete wins
+athlete.win();
+//  action 3:    the athlete trains
+athlete.train();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        audience
+//  action 1:    the audience cheers
+audience.cheer();
+//  action 2:    the audience boos
+audience.boo();
+//  action 3:    the audience listens
+audience.listen();
 //  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        the boss
+//  action 1:    the boss hires
+boss.hire();
+//  action 2:    the boss fires
+boss.fire();
+//  action 3:    the boss inspires
+boss.inspire();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.